src/Event/FeatureToggleNotFoundEvent.php

```diff
@@ -9,7 +9,7 @@
     require __DIR__ . '/../../stubs/event-dispatcher/Event.php';
```

Comment:
Can this not also be removed here?

Reply:
Thank you, don't know how I missed it, it's removed now.

Reply:
No worries! This is why we have PRs!